src/til/ut_til/spscTests.cpp

```diff
@@ -0,0 +1,194 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "precomp.h"
+#include "WexTestClass.h"
+
+using namespace WEX::Common;
+using namespace WEX::Logging;
+using namespace WEX::TestExecution;
+
+struct drop_indicator
+{
+    explicit drop_indicator(int& counter) noexcept :
+        _counter(&counter) {}
+
+    drop_indicator(const drop_indicator&) = delete;
+    drop_indicator& operator=(const drop_indicator&) = delete;
+
+    drop_indicator(drop_indicator&& other) noexcept
+    {
+        _counter = std::exchange(other._counter, nullptr);
+    }
+
+    drop_indicator& operator=(drop_indicator&& other) noexcept
+    {
+        _counter = std::exchange(other._counter, nullptr);
+    }
+
+    ~drop_indicator()
+    {
+        if (_counter)
+        {
+            ++*_counter;
+        }
+    }
+
+private:
+    int* _counter = nullptr;
+};
+
+template<typename T>
+void drop(T&& val)
+{
+    auto _ = std::move(val);
+}
+
+class spscTests
+{
+    BEGIN_TEST_CLASS(spscTests)
+        TEST_CLASS_PROPERTY(L"TestTimeout", L"0:0:10") // 10s timeout

```

Comment:
1. I didn't realize that was a part of TAEF either.
2. I don't think it would 'fix' the CI as such. I think the timeout would still count as a fail. But maybe an actual fail is better than a hang.

Reply:
I mean I assume it works, because that property is being described here: https://docs.microsoft.com/en-us/windows-hardware/drivers/taef/taef-timeouts
Honestly, I haven't tested it yet though.